drivers/lora/Kconfig.sx12xx

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (c) 2019 Manivannan Sadhasivam
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+DT_COMPAT_SX1276 = semtech,sx1276
+DT_COMPAT_SX1261 = semtech,sx1261
+DT_COMPAT_SX1262 = semtech,sx1262
+
+menuconfig LORA_SX12XX
+	bool "Semtech SX-series driver"
+	select HAS_SEMTECH_RADIO_DRIVERS
+	select HAS_SEMTECH_LORAMAC
+	depends on SPI
+	help
+	  Enable LoRa driver for Semtech SX12xx.
+
+if LORA_SX12XX
+
+choice
+	prompt "LoRa Radio chipset"
+	default LORA_SX1276 if $(dt_compat_enabled,$(DT_COMPAT_SX1276))
+	default LORA_SX126X if $(dt_compat_enabled,$(DT_COMPAT_SX1261))
+	default LORA_SX126X if $(dt_compat_enabled,$(DT_COMPAT_SX1262))
+	help
+	  Antenna connection type.
+
+config LORA_SX1276
+	bool "Semtech SX1276 driver"
+	select HAS_SEMTECH_SX1276
```

Comment:
These selects also seem strange. Why are they necessary?

In software, you can just test if any device with the right compatible is enabled using the devicetree API.

Reply:
See comment above about LoRaMAC-node.

Reply:
OK